### PR TITLE
Breadth first

### PR DESCRIPTION
**Pull request by MarikoAlvarado**
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [1/2] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ \x] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

### CODE REVIEW

File: javascript/code-challenges/tree/README.md [hunk 1]
```diff
@@ -20,3 +56,4 @@ I took the approach of creating an add method with and O(h) for time and O(1) fo
 `preOrder()`
 
 
+
```
Comment (adrianhuebner) [inline javascript/code-challenges/tree/README.md:59]: Your code looks really great! Once you have some tests added, I will regrade for you!
